chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestDefinition.java

```diff
@@ -98,9 +117,9 @@ public Builder withValidators(List<Class<? extends ResponseValidator>> validator
         }
 
         /**
-         * As one may expect, creates the object of {@link JLoadTest} type with custom parameters.
+         * Creates the object of JTestDefinition type with custom parameters.
```

Comment:
Why did you remove {@link}?
without it once you rename the referenced class it will not be reflected in a comment